contributing/validator-release-schedule.md

```diff
@@ -0,0 +1,22 @@
+# Validator Release Schedule
+
+-   [Release Freezes](#release-freezes)
```

Comment:
This link seems unnecessary given the short doc. The section is likely visible on the screen already.

Reply:
Good point, removed!